### ISSUE TITLE
Any question .. Spring boot integration Nacos registry

### ISSUE THREAD

**Issue by CjqDy** [alibaba/nacos#4330]
spring boot 2.1.5.RELEASE + nacos-config-spring-boot 0.2.7 + nacos-discovery-spring-boot 0.2.7
in application.yaml
`nacos:
  config:
    server-addr: 10.122.149.207:8848
    namespace: 39681c5a-0edb-49d3-8fc4-6ee3b8d1ea8e
  discovery:
    server-addr: 10.122.149.207:8848
    register:
      service-name: demo-chengjq3
    auto-register: true
    namespace: 41a1c43b-07c0-4307-9e47-cc6c5dd6535e`

config and register has different namespace,but in actually  , service register namespace alse use config namespace
in other words ,register namespace  41a1c43b-07c0-4307-9e47-cc6c5dd6535e invalid

**Comment by paderlol** [comment]
Nacos itself supports different namespaces, you need to mention the Spring Boot Nacos community in this issue

**Comment by CjqDy** [comment]
> Nacos itself supports different namespaces, you need to mention the Spring Boot Nacos community in this issue
ok, thanks
